### DISCUSSION TITLE
开源之道·收入：适兕是如何生存的？ - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#86] [Announcements]
# 开源之道·收入：适兕是如何生存的？ - 「开源之道」

https://opensourceway.community/posts/the_way_of_open_source/how-to-make-money-of-open-source-way/

**Comment by ZoomQuiet** [comment]
#是也乎

> 这是一个最好的时代, 也是一个最坏的时代

其实吧, 嘦一个人想通了, 任何生活都是可以接受而不是忍受,
否则, 任何一种生活都只能忍受, 而无法习惯...

19年在上海首次当面和适兕交流时, 
就想拉蟒营®里来担当课程讲师,
通过收费来支撑和拓展开源理解;

不过, 显然 适兕 有自己的节奏,
为大众多读书, 读好书, 然后, 开放自己的调用接口,
接受所有和开源相关的咨询服务,
将自己用另外一种形式, 编译成为社会必要知识节点,

善.

**Reply by lijiangsheng1** [reply]
现代社会，总还是有那么一点给人留活口的生存空间的。能用尽全力活着就是件很幸运的事了。
